tasks/extract_filepaths_for_granule/bin/build.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
```

Comment:
Recommend including the schemas in the build.

Reply:
not sure about this. Can you clarify or show an example?

Reply:
copy_to_glacier_cumulus_translator does this well. Basically, the way things are now, nothing is actually checked against your schema files.

Reply:
ok gotcha. will add that schema part to `bin/build.sh`